tst/testinstall/methwhy.tst

```diff
@@ -1,9 +1,11 @@
-gap> START_TEST("methwy.tst");
+gap> START_TEST("methwhy.tst");
 
 #
 gap> ApplicableMethod();
+#I  ApplicableMethod requires at least two arguments
 Error, usage: ApplicableMethod(<opr>,<arglist>[,<verbosity>[,<nr>]])
 
+
```

Comment:
Why is there an extra empty line now? Is this an accident, or does `ApplicableMethod` really print it? Either variant seems like a bug (or misfeature) to me, though... :/